qemu/tests/hv_crash.py

```diff
@@ -0,0 +1,68 @@
+import logging
+
+from virttest import error_context
+from virttest import env_process
+from virttest import utils_misc
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test the hv_crash flag avaliability
+
+    1) boot the guest with hv_crash flag
+    2) use nmi to make the guest crash, the qemu process should quit
+    3) boot the guest without hv_crash flag
+    4) use nmi again, the qemu should not quit, the guest will reboot
+    5) after reboot, the guest should work well
+
+    param test: the test object
+    param params: the test params
+    param env: the test env object
+    """
+
+    def _boot_guest_with_cpu_flag(hv_flag):
+        """
+        Boot the guest, with param cpu_model_flags set to hv_flag
+
+        param hv_flag: the hv flags to set to cpu
+
+        return: the booted vm
+        """
+        params["cpu_model_flags"] = hv_flag
+        params["start_vm"] = "yes"
+        vm_name = params["main_vm"]
+        env_process.preprocess_vm(test, params, env, vm_name)
+        return env.get_vm(vm_name)
+
+    def _trigger_crash(vm):
+        """
+        Trigger a system crash by nmi
+        """
+        session = vm.wait_for_login(timeout=timeout)
+        session.cmd(set_nmi_cmd)
+        vm.monitor.nmi()
+
+    timeout = params.get("timeout", 360)
+    flags_without_hv_crash = params["flags_without_hv_crash"]
+    flags_with_hv_crash = params["flags_with_hv_crash"]
+    set_nmi_cmd = params["set_nmi_cmd"]
+
+    error_context.context("Boot the guest with hv_crash flag", logging.info)
+    vm = _boot_guest_with_cpu_flag(flags_with_hv_crash)
+
+    error_context.context("Make the guest crash", logging.info)
+    _trigger_crash(vm)
+    logging.info("Check the qemu process is quit")
+    if not utils_misc.wait_for(vm.is_dead, 10, 1, 1):
```

Comment:
`if not utils_misc.wait_for(vm.is_dead, 10, 1, 1):` -> `if not utils_misc.wait_for(vm.is_dead(), 10, 1, 1):`
The vm instance doesn't have the `is_dead` method.

Reply:
I checked the method, it seems that should use utils_misc.wait_for(vm.is_dead, 10, 1, 1).

Reply:
Sorry, I mistake it. you are right.